linode/manager
# M3-5310: Organize Storybook components according to new outline

## Description

Organize the existing stories into folders and removed a couple stories that are no longer relevant.

~Wait until #7749 is merged and decide how to organize `EventListItem` and `ScriptCode`.~

## How to test

- Run `yarn storybook`
- Navigate to `http://localhost:6006/` and check out the new layout
